applicationconfig/client/pom.xml

```diff
@@ -5,86 +5,93 @@
 -->
```

Comment:
The indentation on this was 4 spaces, but our .editorconfig and all the other ones are 2 spaces. So this incorporated that change.